themes-default/slim/static/js/templates/app-header.vue

```diff
@@ -13,24 +13,26 @@
         </div>
         <div v-if="auth.isAuthenticated" class="collapse navbar-collapse" id="main_nav">
             <ul class="nav navbar-nav navbar-right">
-                <li id="NAVhome" class="navbar-split dropdown" :class="{ active: topMenu === 'home' }">
-                    <app-link href="home/" class="dropdown-toggle" aria-haspopup="true" data-toggle="dropdown" data-hover="dropdown"><span>Shows</span>
-                    <b class="caret"></b>
+                <li @mouseover="toggleDropdown('home')" @mouseout="toggleDropdown('home')" id="NAVhome" class="navbar-split dropdown" :class="{ active: topMenu === 'home' }">
+                    <app-link href="home/" class="dropdown-toggle" aria-haspopup="true" data-toggle="dropdown" data-hover="dropdown">
+                        <span>Shows</span> <b class="caret"></b>
                     </app-link>
-                    <ul class="dropdown-menu">
-                        <li><app-link href="home/"><i class="menu-icon-home"></i>&nbsp;Show List</app-link></li>
-                        <li><app-link href="addShows/"><i class="menu-icon-addshow"></i>&nbsp;Add Shows</app-link></li>
-                        <li><app-link href="addRecommended/"><i class="menu-icon-addshow"></i>&nbsp;Add Recommended Shows</app-link></li>
-                        <li><app-link href="home/postprocess/"><i class="menu-icon-postprocess"></i>&nbsp;Manual Post-Processing</app-link></li>
-                        <template v-if="recentShows.length > 0">
-                        <li role="separator" class="divider"></li>
-                        <li v-for="recentShow in recentShows" :key="recentShow.link">
-                            <app-link :href="recentShow.link">
-                                <i class="menu-icon-addshow"></i>&nbsp;{{ recentShow.name }}
-                            </app-link>
-                        </li>
-                        </template>
-                    </ul>
+                    <transition name="fade" :duration="500">
```

Comment:
Once you add the transition effect to CSS, you can remove the `duration` property, It's optional.
```css
.fade-enter-active, .fade-leave-active {
    transition: opacity .5s;
}

.fade-enter, .fade-leave-to {
    opacity: 0;
}
```